Scripts/Runtime/Entities/TaskDriver/TaskSet/Job/JobConfig/AbstractJobConfig.cs

```diff
@@ -94,6 +94,7 @@ public override string ToString()
         /// <inheritdoc cref="IJobConfig.RunOnce"/>
         public IJobConfig RunOnce()
         {
+            IsEnabled = true;
```

Comment:
Logger related?

Reply:
> Tag-Alongs
> 
>     Made the RunOnce function for JobConfigs able to be called after configuration in response to structural changes.
>     Updated docs accordingly.

But your comments about the logger are making me close this PR and do a different one

Reply:
Oops, yea sorry missed that.